Makefile

```diff
@@ -15,27 +16,27 @@ e2e_build_images: e2e_clean_slate
 	@docker build -t ethereum:prebuilt -f integration_tests/ethereum/Dockerfile integration_tests/ethereum/
 
 e2e_clean_slate:
-	@./clean_slate.sh
+	@scripts/clean_slate.sh
 
 e2e_cork_test: e2e_aave_v2_stablecoin_test e2e_cellar_v1_test e2e_cellar_v2_test e2e_proposal_test
 
-# Because of the way `make` works, using the e2e_clean_slate target twice as prerequisites
-# for the individual tests doesn't work when `e2e_cork_test` runs the test targets in series,
+# Because of the way `make` works, using the e2e_clean_slate as as a prerequisite for
+# the individual tests doesn't work when `e2e_cork_test` runs the test targets in series,
 # so we explicitly call the cleanup script in each test target.
-e2e_aave_v2_stablecoin_test:
-	@./clean_slate.sh
+e2e_aave_v2_stablecoin_test: protos
```

Comment:
Why are we building protos every time we run a test? This should only have to happen at dev time.

Reply:
Since I moved the rust proto build into the `build.rs` file which executes every build, I figured it the same spirit I would make this change. It may be unnecessary but seemed like a good idea at the time.

Reply:
I suppose, but this means CI will be building protos, and if the build introduces file changes that weren't committed, CI results won't match the checked-in code.